Doc/library/configparser.rst

```diff
@@ -990,6 +996,13 @@ ConfigParser Objects
    converter gets its own corresponding :meth:`!get*` method on the parser
    object and section proxies.
 
+   When *allow_unnamed_section* is ``True`` (default: ``False``),
+   the first section name can be omitted, options with no section name
+   specified ahead (often called ``"Global Options"`` or
+   ``options in "Default Section"``) are allowed. See
+   `"Supported INI File Structure" section<#supported-ini-file-structure>`_
+   for details.
```

Comment:
I think drop the last part of the first sentence. Nowhere else mentions _global options_, and _default section_ is a different thing.

Suggest linking directly to the Unnamed Sections section. Maybe spacing the link target from its text might fix the compilation error too.

```suggestion
   When *allow_unnamed_section* is ``True`` (default: ``False``),
   the first section name can be omitted. See the
   `"Unnamed Sections" section <#unnamed-sections>`.
```

Reply:
Thanks!

The only fly in the ointment is the missing `_` to after `<#unnamed-sections>`